docs/sources/tutorials/get-started.md

```diff
@@ -222,7 +255,7 @@ sudo systemctl reload alloy
 Open [http://localhost:12345] and click the Graph tab at the top.
 The graph should look similar to the following:
 
-{{< figure src="/media/docs/alloy/tutorial/healthy-config.png" alt="Logs reported by Alloy in Grafana" >}}
+{{< figure src="https://github.com/grafana/alloy/assets/60980933/f68cc9e1-96e9-4ca3-b594-b7bb250314e4" alt="Logs reported by Alloy in Grafana" >}}
```

Comment:
This is the wrong place for images.  Refer to: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#where-to-store-media-assets

If there's a new image for this tutorial, it can be uploaded to: https://admin.grafana.com/upload/media/docs/alloy/tutorial/ and linked (refer to the Writer's Toolkit for info about how to do the linking).

Reply:
Thank you so much for all the great changes! I uploaded the image from the correct media storage. I think I did it, right but please do take a look.